client/src/featureform/resources.py

```
@@ -2059,6 +2059,14 @@ def proto(self) -> pb.Columns:
             ts=self.timestamp,
         )
 
+    def to_entity_mappings_proto(self, entity_name: str) -> pb.EntityMappings:
```

Ruff (0.8.2)

`pb` may be undefined, or defined from star imports

(F405)